src/logger/loggerConfig.c

```diff
@@ -268,8 +268,14 @@ int encodeSampleRate(int sampleRate)
 {
 
     switch(sampleRate) {
+#if MAX_SENSOR_SAMPLE_RATE > 100
+    case 1000:
+        return SAMPLE_1000Hz;
+    case 500:
+        return SAMPLE_500Hz;
     case 200:
         return SAMPLE_200Hz;
+#endif
```

Comment:
Might it be simpler to have a MAX_SAMPLE_RATE variable that says simply something like

```c
if (sampleRate > MAX_SAMPLE_RATE)
    return SAMPLE_DISABLED;
```

Then you don't put macros in code and that makes it cleaner.  Could also return SAMPLE_MAX_RATE instead of SAMPLE_DISABLED too.